src/traversals/parallel_bfs.jl

```diff
@@ -15,7 +15,7 @@ import Base: push!, shift!, isempty, getindex
 
 export bfs_tree, LevelSynchronousBFS
 
-mutable struct LevelSynchronousBFS <: AbstractGraphVisitAlgorithm end
+struct LevelSynchronousBFS end
```

Comment:
I don't know. This is Rohit's code; I just found it funny that it was mutable with no data fields.